types/mapbox-gl/index.d.ts

```diff
@@ -35,7 +39,7 @@ declare namespace mapboxgl {
 
 		getBounds(): mapboxgl.LngLatBounds;
 
-		setMaxBounds(lnglatbounds?: mapboxgl.LngLatBounds | number[][]): this;
+		setMaxBounds(lnglatbounds?: LngLatBoundsLike): this;
```

Comment:
Is this type change intentional? Because the input type of this function changed from type ```mapboxgl.LngLatBounds | number[][]``` to ```number[][] | LngLatLike[] | LngLatBounds```